docs/csv/example_files/ballot_style.txt

```diff
@@ -1 +1,3 @@
-id,image_uri,ordered_contest_ids,party_ids
\ No newline at end of file
+id,image_uri,ordered_contest_ids,party_ids
+bs00010,http://i.giphy.com/26BoCh3PgT8ai45ji.gif,oc2025,par02
+bs00011,http://i.giphy.com/3oEjHYDWEICgEpAOjK.gif,oc3000,par01
```

Comment:
I think I only just now loaded these links.

*sigh*

Reply:
These are great.  Adding them to our collection.